packages/halo-theme/src/custom-elements/ef-datetime-picker.less

```diff
@@ -76,4 +76,11 @@
     border-color: @input-hover-border-color;
     color: @input-hover-text-color;
   }
+
+  [part~='button-clear']:hover {
+    color: @input-focused-border-color;
+  }
+  [part~='button']:active {
```

Comment:
`:active` state of the clear button is invisible.

Reply:
Fixed.